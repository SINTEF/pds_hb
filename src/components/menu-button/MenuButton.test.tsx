import React from 'react'
import renderer from 'react-test-renderer'
import { MenuButton } from '.'

it('renders without crashing', () => {
  const tree = renderer.create(<MenuButton label="Test button" />).toJSON()
  expect(tree).toMatchSnapshot()
})
