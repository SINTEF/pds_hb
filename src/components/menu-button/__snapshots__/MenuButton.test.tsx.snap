// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="container"
>
  <hr
    className="line"
  />
  <button
    className="button standard "
  >
    Test button
  </button>
</div>
`;
